src/easier68k/core/enum/ea_mode_bin.py

```diff
@@ -7,31 +7,47 @@
 
 class EAModeBinary:
```

Comment:
We should make this an `Enum` using the actual class

Reply:
I started to, but I think in this case it will make the code only more complicated because I need to be able to freely convert between and compare the enum and it's value. It will require quite a bit of conversion. Also, it's never used outside the file.

Reply:
Or actually IntEnum might work.

Reply:
IntEnum worked